### ISSUE TITLE
Documentation: priceRanges example has two "View HTML" buttons

### ISSUE THREAD

**Issue by pixelastic** [algolia/instantsearch.js#505]
The priceRanges example has two "View HTML" buttons

![image](https://cloud.githubusercontent.com/assets/283419/11029739/43e84bb0-86c9-11e5-8986-362c94555bc0.png)

**Comment by pixelastic** [comment]
Ok, I found where it come from. The javascript code checks for all `pre` that contains `#widgetName`, and it got confused by the `price` and `priceRange` widget.

```javascript
var buttons = $('.code-box pre:contains("#' + id + '")').closest('.code-box').find('.btn-group');
```

I tried fixing it by wrapping each widget in its own container, in a way to scope the selectors, but it messed the display.
I instead tried to "fix" the selector to search for `'#widget-name'` (including quotes), but then it got confused by the weird `#priceranges.widget-container` selector of the example.
I then tried to rename the selector to `#priceranges` to stay consistent and I realized that I got conflicts with other elements (headings) having the same id in the page...

In the end I gave up, I'll tackle other issues first, but this one needs quite a code cleanup.
